final-project/Figures/api-component.mmd

```diff
@@ -0,0 +1,20 @@
+C4Container
+title Component diagram for APIProxy for OpenTogetherTube
+
+    Person(customer, Customer, "A user of OTT", $tags="v1.0")
+
+    Container_Boundary(c1, "OpenTogetherTube") {
+    Container(browser, "Browser", "Web Browser", "Used by the user to access the application", "v1.0")
+    Container(server, "Server", "Java", "Serves as the backend for the OpenTogetherTube platform", "v1.0")
```

Comment:
The server is not written in Java. It's written in mostly Typescript, and a little Javascript.